packages/asc-ui/src/theme/CompactThemeProvider.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import { storiesOf } from '@storybook/react'
+import Paragraph from '../components/Paragraph'
+import Heading from '../components/Heading'
+import CompactThemeProvider from './CompactThemeProvider'
+
+storiesOf('Atoms/Typography/CompactThemeProvider', module).add(
```

Comment:
CompactThemeProvider isn't really a component / part of Typography... Consider deleting this?

Reply:
ok